### ISSUE TITLE
Error when choosing remote version when the local and remote versions differ

### ISSUE THREAD

**Issue by smathot** [dschreij/opensesame-extension-osf#6]
If I open an experiment that is linked to the OSF, but for which the local and remote versions differ, I can choose which version I want to keep. If choose the remote version, than I can choose whether I want to create a backup of the local copy or not. So far so good, but then I get the following crash.

~~~
Traceback (most recent call last):
  File "/home/sebastiaan/.local/lib/python2.7/site-packages/QOpenScienceFramework/manager.py", line 380, in <lambda>
    callback, *args, **kwargs
  File "/home/sebastiaan/.local/lib/python2.7/site-packages/QOpenScienceFramework/manager.py", line 839, in __reply_finished
    callback(reply, *args, **kwargs)
  File "/home/sebastiaan/.local/lib/python2.7/site-packages/QOpenScienceFramework/manager.py", line 938, in __download_finished
    fcb(reply, *args, **kwargs)
  File "/home/sebastiaan/.local/share/opensesame_extensions/OpenScienceFramework/OpenScienceFramework.py", line 1383, in __experiment_downloaded
    self.openingDialog.hide()
AttributeError: 'OpenScienceFramework' object has no attribute 'openingDialog'
~~~

**Comment by dschreij** [comment]
Alright,  thanks. That will have to wait until after I'm back.

**Comment by smathot** [comment]
Of course, enjoy your vacation!

**Comment by dschreij** [comment]
Fixed in e5274a2e26d232b103c5de643a00f85e088e8108 and v1.0.8
